lib/config/presets/gitlab.ts

```diff
@@ -4,12 +4,11 @@ import { Preset } from './common';
 
 const { get: glGot } = api;
 
-const GitLabApiUrl = 'https://gitlab.com/api/v4/projects';
-
 async function getDefaultBranchName(
-  urlEncodedPkgName: string
+  urlEncodedPkgName: string,
+  endpoint: string
 ): Promise<string> {
-  const branchesUrl = `${GitLabApiUrl}/${urlEncodedPkgName}/repository/branches`;
+  const branchesUrl = `${endpoint}/projects/${urlEncodedPkgName}/repository/branches`;
```

Comment:
Does this work ok if `endpoint` is defined with a trailing `/` ?

Reply:
Nope, thanks! Fixed with dbb0debbacff04c9ad7bf1508aa02e151fecf567